### DISCUSSION TITLE
Measured points in QFIELDS

### DISCUSSION THREAD

**Discussion by SaSiGS** [opengisch/QField#5175] [Q&A]
Hi,

i'm new to QFIELDS and searched the manuals and the board but couldn't find a match for my questions. Maybe i'm just not using the best keywords (not my first language). 
i'm trying to find out, if i can see my measured points (supported by a GPS device) as a list INSIDE of QFIELDS. It's no problem to get the xyz and a few more prespecified informations in the synced project in QGIS. But i need to be able to review these outside in the field.
And with thousands of measurements in (sometimes very) tiny areas it proves to be very difficult to find the exact point i need.
So first question: is there some kind of filter function for the points or are we still talking of "just gather, review later"?

Second question: If i create polygons, are the base points stored in a way, that i can export both polygon and every single point i used to draw it separately? Or do i have to create the points through QGIS (which is probably not enough, since i have to able to get z, not just lon and lat?

Thanks for your help
SaSiGS

**Comment by SeqLaz** [comment]
QField does support external receivers for better measurements of heights calculation with increased accuracy. You can find more information on how to use an external GNSS receiver in our documentation: [Using an External GNSS Receiver](https://docs.qfield.org/how-to/gnss/#using-an-external-gnss-receiver).

As for filtering features, this can be achieved using the search bar. You can preconfigure how features are displayed and sorted:
- [Searching in QField](https://docs.qfield.org/how-to/search/)
- [Sorting Layer Features](https://docs.qfield.org/how-to/map-interaction/#sort-layer-features)

Regarding your last question, QField does not currently support the use of core plugins for directly extract the vertices. However, if you prefer not to digitize manually, you can utilize the core plugin "Extract Vertices" as an alternative solution in QGIS.
